bin/manager_start.sh

```diff
@@ -59,7 +59,11 @@ if [ -n "$JOBSERVER_KEYTAB" ]; then
   SPARK_SUBMIT_OPTIONS="$SPARK_SUBMIT_OPTIONS --keytab $JOBSERVER_KEYTAB"
 fi
 
-cmd='$SPARK_HOME/bin/spark-submit --class $MAIN --driver-memory $JOBSERVER_MEMORY
+if [ -z "${SJS_LAUNCHER}" ]; then
```

Comment:
do you think we could avoid those changes by exporting SPARK_HOME to resources/spark-launcher?

Reply:
I thought about but decided not to do that for two reasons
- it's not really a SPARK_HOME, someone/something may fail because of that, now or in the future.  Having launcher explicitly specified is an attempt to keep things clear.
- later down the road SPARK_HOME is set with meaningful value, I want to avoid errors that are caused by transient value

Reply:
becuase of 2, 1 should not happen :)